src/main/kotlin/ro/hibyte/betting/entity/Event.kt

```diff
@@ -0,0 +1,71 @@
+package ro.hibyte.betting.entity
+
+import com.fasterxml.jackson.databind.BeanDescription
+import jakarta.persistence.*
+import java.sql.Timestamp
+
+@Entity
+data class Event(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    var eventId: Long = 0,
+    var name: String = "",
+    var description: String = "",
+    var creator:String = "",
+    var template:String = "",
+    @ElementCollection
+    var tags: List<String> = emptyList(),
+    @OneToMany
+    @JoinColumn(name  = "eventId")
+    var betTypes:List<BetType> = emptyList(),
+    @OneToMany
+    @JoinColumn(name = "eventId")
+    var bets:MutableList<Bet> = arrayListOf(),
+    @OneToMany
+    @JoinColumn(name = "eventId")
+    var users:List<UserProfile> = emptyList(),
+    @ElementCollection
+    var userGroups : List<String> = emptyList(),
+    @ElementCollection
+    var userProfiles : List<Long?> = emptyList(),
+    var created : Timestamp = Timestamp(System.currentTimeMillis()),
+    var lastModified :Timestamp = Timestamp(System.currentTimeMillis()),
+    var startsAt :Timestamp = Timestamp(System.currentTimeMillis()),
+    var endsAt :Timestamp = Timestamp(System.currentTimeMillis()),
+    var status: Status = Status.Draft
+) {
+    constructor(
```

Comment:
I don't think this constructor is necessary, because in case you only need a subset of the main constructor's parameters, you can just use something like this:

```kotlin
Event(name = "name", status = Status.Draft)
```

Please remove